tests/queries/0_stateless/02009_from_infile.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+set -e
+
+[ -e "${CLICKHOUSE_TMP}"/test_infile.gz ] && rm "${CLICKHOUSE_TMP}"/test_infile.gz
+[ -e "${CLICKHOUSE_TMP}"/test_infile ] && rm "${CLICKHOUSE_TMP}"/test_infile
+
+echo "Hello" > "${CLICKHOUSE_TMP}"/test_infile
+
+gzip "${CLICKHOUSE_TMP}"/test_infile
+
+${CLICKHOUSE_CLIENT} --query "DROP TABLE IF EXISTS test_infile;"
+${CLICKHOUSE_CLIENT} --query "CREATE TABLE test_infile (word String) ENGINE=Memory();"
+${CLICKHOUSE_CLIENT} --query "INSERT INTO test_infile FROM INFILE '${CLICKHOUSE_TMP}/test_infile.gz' FORMAT CSV;"
+${CLICKHOUSE_CLIENT} --query "SELECT * FROM test_infile;"
+
+# if it not fails, select will print information
+${CLICKHOUSE_LOCAL} --query "CREATE TABLE test_infile (word String) ENGINE=Memory(); INSERT INTO test_infile FROM INFILE '${CLICKHOUSE_TMP}/test_infile.gz' FORMAT CSV; SELECT * from test_infile;" 2>&1  | grep -q "UNKNOWN_TYPE_OF_QUERY" && echo "Correct Local" || echo 'Fail'
```

Comment:
Maybe it should work with clickhouse-local. We do not allow to execute queries like `FROM INFILE` on clickhouse-server, because it's possible to read/write arbitrary files on server-side using such queries. But there are no reasons to disable such queries in clickhouse-local.